### ISSUE TITLE
Keyboard input problem

### ISSUE THREAD

**Issue by MrCoderMD** [hrydgard/ppsspp#19309]
I'm using the latest version of ppsspp on pc right now. However, the problem is while playing tekken 6, whenever I'm trying to do up+forward+any action button(⭕,❌,🟥,⚠️) it's not working. The action buttons are working after up+forward. It happens with both d-pad and analog stick, but I have no problem while doing this with touch controls. What can I do?

**Comment by LunaMoo** [comment]
This is most likely just a limitation of your keyboard, you can buy a different one or map your keys in a way that is possible to input at the same time, it differs per keyboard so you just need to test yourself.

**Comment by anr2me** [comment]
Yeah, i remembered having a keyboard that doesn't worked well when combining up arrow + left (or  was it right) arrow before.
Even this laptop's keyboard i'm currently using can't detects the 3rd button (ie. Z) combination after up+left buttons pressed when i tested it with https://en.key-test.ru/ but worked with X button as the 3rd combination.

So you should try to testing those buttons combination at https://en.key-test.ru/ and see whether it worked properly.
